chore: fix EditorConfig lint errors

ditsus opened the pull request:
Resolves #5423.

## Description

> What is the purpose of this pull request?

The purpose of this pull request is to fix EditorConfig linting issues with spaces and indents in the following files:

lib/node_modules/@stdlib/_tools/eslint/rules/no-nested-require/examples/index.js
lib/node_modules/@stdlib/math/base/special/rsqrt/manifest.json
lib/node_modules/@stdlib/ndarray/base/strides2offset/manifest.json

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #5423

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes. 

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

ditsus committed:
chore: fix EditorConfig lint errors

ditsus committed:
chore: fix EditorConfig lint errors

stdlib-bot committed:
docs: update namespace table of contents

PR-URL: https://github.com/stdlib-js/stdlib/pull/5533
Reviewed-by: Athan Reines <kgryte@gmail.com>

yugalkaushik committed:
bench: suppress uninitialized variable warning

PR-URL: https://github.com/stdlib-js/stdlib/pull/5523
Closes: https://github.com/stdlib-js/stdlib/issues/5503
Reviewed-by: Athan Reines <kgryte@gmail.com>

anandkaranubc committed:
refactor: move implementations and macros to separate files

PR-URL: https://github.com/stdlib-js/stdlib/pull/5540
Reviewed-by: Athan Reines <kgryte@gmail.com>
Co-authored-by: stdlib-bot <noreply@stdlib.io>

stdlib-bot committed:
docs: update namespace TypeScript declarations

PR-URL: https://github.com/stdlib-js/stdlib/pull/5531
Reviewed-by: Athan Reines <kgryte@gmail.com>

anandkaranubc committed:
refactor: move implementations and macros to separate files

PR-URL: https://github.com/stdlib-js/stdlib/pull/5541
Reviewed-by: Athan Reines <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/_tools/eslint/rules/no-nested-require/examples/index.js:
```diff
@@ -36,17 +36,17 @@ result = linter.verify( code, {
+			'endLine': 2,
+			'endColumn': 29
+		}
+]
```
```suggestion
	]
```

kgryte commented on lib/node_modules/@stdlib/math/base/special/rsqrt/manifest.json:
```diff
@@ -1,78 +1,78 @@
-			]
-		}
-	]
+    "options": {
```
Per our EditorConfig configuration, please use two-space indentation in all JSON files. See https://github.com/stdlib-js/stdlib/blob/183d056687e023d43cded450f0d9e4739aaff8bd/.editorconfig#L46.

kgryte commented on lint_editorconfig_errors.txt:
Please don't include files such as this.

kgryte replied:
This file should be deleted.

anandkaranubc committed:
refactor: move implementations and macros to separate files

PR-URL: https://github.com/stdlib-js/stdlib/pull/5544
Reviewed-by: Athan Reines <kgryte@gmail.com>

anandkaranubc committed:
feat: add `DIII_D` macro in `math/base/napi/quaternary`

PR-URL: https://github.com/stdlib-js/stdlib/pull/5545
Reviewed-by: Athan Reines <kgryte@gmail.com>

anandkaranubc committed:
docs: fix typo by replacing `int_32` with `int32_t`

PR-URL: https://github.com/stdlib-js/stdlib/pull/5546
Reviewed-by: Athan Reines <kgryte@gmail.com>
Co-authored-by: stdlib-bot <noreply@stdlib.io>

saurabhraghuvanshii committed:
bench: refactor random number generation

PR-URL: https://github.com/stdlib-js/stdlib/pull/5397

Reviewed-by: Karan Anand <119553199+anandkaranubc@users.noreply.github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

anandkaranubc committed:
feat(stats): add C implementation for `stats/base/dists/hypergeometric/skewness`

PR-URL: https://github.com/stdlib-js/stdlib/pull/4560
Ref: https://github.com/stdlib-js/stdlib/issues/3666

Co-authored-by: stdlib-bot <noreply@stdlib.io>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

anandkaranubc committed:
feat(stats): add C implementation for `stats/base/dists/hypergeometric/stdev`

PR-URL: https://github.com/stdlib-js/stdlib/pull/4559
Closes: https://github.com/stdlib-js/stdlib/issues/3667

Co-authored-by: stdlib-bot <noreply@stdlib.io>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>
Signed-off-by: Karan Anand <119553199+anandkaranubc@users.noreply.github.com>

Planeshifter committed:
build: do not run workflow for PRs by bots

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: na
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: na
  - task: run_c_benchmarks
    status: na
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: na
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: na
---

dependabot[bot] committed:
build(deps): bump ossf/scorecard-action from 2.4.0 to 2.4.1

PR-URL: #5539

Signed-off-by: dependabot[bot] <support@github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

dependabot[bot] committed:
build(deps): bump msys2/setup-msys2 from 2.26.0 to 2.27.0

PR-URL: #5535

Signed-off-by: dependabot[bot] <support@github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

dependabot[bot] committed:
build(deps): bump r-lib/actions from 2.11.1 to 2.11.2

PR-URL: #5538

Signed-off-by: dependabot[bot] <support@github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

dependabot[bot] committed:
build(deps): bump actions/upload-artifact from 4.6.0 to 4.6.1

PR-URL: #5537

Signed-off-by: dependabot[bot] <support@github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

dependabot[bot] committed:
build(deps): bump actions/cache from 4.2.0 to 4.2.2

PR-URL: #5536

Signed-off-by: dependabot[bot] <support@github.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>

yuvimittal committed:
feat: add C implementation for `stats/base/dists/planck/entropy`

PR-URL: https://github.com/stdlib-js/stdlib/pull/4898
Closes: https://github.com/stdlib-js/stdlib/issues/4897

Co-authored-by: Philipp Burckhardt <pburckhardt@outlook.com>
Reviewed-by: Philipp Burckhardt <pburckhardt@outlook.com>
Signed-off-by: Philipp Burckhardt <pburckhardt@outlook.com>

ditsus committed:
chore: fix EditorConfig linting errors

ditsus committed:
chore: fix EditorConfig linting errors

ditsus committed:
chore: fix EditorConfig linting errors

ditsus committed:
chore: fix EditorConfig linting errors

ditsus committed:
chore: fix EditorConfig linting errors
